### DISCUSSION TITLE
the ability to get the real address of any user

### DISCUSSION THREAD

**Discussion by nigel77775** [farcasterxyz/protocol#172] [Q&A] [answered]
Some method from https://client.warpcast.com/v2/... reveals real evm addresses of users. You only need to know the fid, I’m not entirely sure that it should work like that, it seems to me that this is a security vulnerability - Broken Access Control. Maybe I just didn't fully understand it, and this is normal behavior...

**Comment by horsefacts** [comment, marked as answer]
This is by design. Farcaster is a public social network and [Verification messages](https://github.com/farcasterxyz/protocol/blob/main/docs/SPECIFICATION.md#25-verifications) are public on Hubs. You don't have to create any verifications to use Farcaster: If you don't want this information to be public, don't publish it to your profile.
